.github/workflows/sdk-gen.yml

```diff
@@ -5,23 +5,26 @@
 
 name: Data Management System SDK Generation
 on:
-  release:
-    types:
-      - prereleased
-
+  pull_request:
+    branches:
```

Comment:
I will remove every thing in this file once all changes are good with reviewer testing